openpype/hosts/aftereffects/api/ws_stub.py

```diff
@@ -284,7 +284,7 @@ def get_item(self, item_id):
 
         return None
 
-    def import_file(self, path, item_name, import_options=None):
+    def import_file(self, path, item_name, import_options=None, fps=None):
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function